software/bkeymaps.txt

```diff
@@ -0,0 +1,29 @@
+SETTING THE KEYBOARD LAYOUT
+________________________________________________________________________________
+
+
+Acquiring keymaps from the bkeymaps package
+________________________________________________________________________________
+
+You can get the bkeymaps package from Community.
+
++------------------------------------------------------------------------------+
+|                                                                              |
+|   $ kiss b bkeymaps                                                          |
+|   $ kiss i bkeymaps                                                          |
+|                                                                              |
++------------------------------------------------------------------------------+
```

Comment:
```
Begin by first verifying that you have bkeymaps installed:

+------------------------------------------------------------------------------+
|                                                                              |
|   $ kiss b dhcpcd && kiss i dhcpcd                                           |
|                                                                              |
+------------------------------------------------------------------------------+
```

Reply:
How about "First, install the bkeymaps package from the Community repository" to combine it with the previous comment?

Reply:
```
Begin by first verifying that you have bkeymaps installed from the Community repository:
```

Reply:
Isn't "Begin by first" redundant?

Reply:
:+1: